Makefile

```diff
@@ -19,12 +19,12 @@ release: translations
 	dch -i
 
 dist: orig
-	dpkg-buildpackage -S
+	dpkg-buildpackage -S -us -uc
```

Comment:
It can do both at the same time??? This must be heaven :angel: :partly_sunny: :fireworks: